classes/local/object_manipulator/manipulator.php

```diff
@@ -27,6 +27,7 @@
 
 use dml_exception;
 use stdClass;
+use tool_objectfs\local\manger as config;
```

Comment:
Why are you aliasing this as config?

config::update_object doesnt make much sense vs

manager::update_object